src/jmh/java/column/store/input/CreateInput.java

```diff
@@ -0,0 +1,102 @@
+package column.store.input;
+
+import column.store.api.column.Column;
+import com.devskiller.jfairy.Fairy;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Random;
+
+/**
+ * Creation of a file for {@link column.store.input.InMemoryReader}.
+ */
+@SuppressWarnings("checkstyle:MagicNumber")
+
+public final class CreateInput {
+
+    private static final int NUMBER_OF_RECORDS = 1_000_000;
+    private static final Random RANDOM = new Random();
+
+    static {
+        long seed = RANDOM.nextLong();
+        System.out.println("Using seed: " + seed);
+        RANDOM.setSeed(seed);
+    }
+
+    public static void main(final String[] args) throws IOException {
+        var name = Column.forString("name");
+        var male = Column.forBoolean("male");
+        var age = Column.forLong("age");
+        var height = Column.forDouble("height");
+        var id = Column.forId("id");
+
+        var testDir = Path.of("testDir");
+        deleteRecursively(testDir);
```

Comment:
is testdir currently a placeholder ?
would rather put it into f.e.: 'src/test/resources'

Reply:
We can change that 😄

Reply:
👍